source/healthBar.js

```diff
@@ -5,10 +5,12 @@ export class HealthBar {
 	x = 0;
 	y = 0;
 	height = 0;
+    width = 0;
```

Comment:
tab

Reply:
Newest commit converted tabs to spaces (like most ides do automatically) so git will parse it properly. Should be fine from now on